docs/current/sailbot_workspace/launch_files.md

```diff
@@ -0,0 +1,182 @@
+The process is as follows:
+
+1. The package launch file is invoked with the user passing arguments via the CLI and specifying a configuration file.
+2. Global argument declarations and environment variables are loaded into the launch process.
+3. Local arguments, specific to the package, are declared.
+4. Both global and local arguments are parsed based on the argument declarations and are set for use upon start up.
+5. The ROS nodes belonging to the package begin execution, utilizing the ROS parameters from the configuration file.
+
+??? warning "When launching individual packages, be aware of dependencies between ROS packages"
+    Some packages rely on the data produced by other packages in the system. This may cause only
+    partial functionality of the ROS node(s) that are running inside the launched package. Therefore,
+    it may be necessary to launch multiple packages manually to get the desired functionality.
+
+### The Global Launch Process
+
+<!-- This was generated by going to the source, selecting File > Embed > iFrame, and deselecting Layers, Tags, and Edit -->
+<!-- Source: https://drive.google.com/file/d/19_QAeRoTDw4qQRp6w6It4_d-8YmKw-ih/view?usp=sharing -->
+<iframe frameborder="0" style="width:100%;height:600px;" src="https://viewer.diagrams.net/?highlight=0000ff&nav=1&title=Global_Launch_Architecture.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D19_QAeRoTDw4qQRp6w6It4_d-8YmKw-ih%26export%3Ddownload"></iframe>
+
+The global launch process is intended to start up the entire system (both the development and production environments).
+This process invokes the package launch files for each ROS package used in the system through a global launch file.
+The process is as follows:
+
+1. The global launch file is invoked with the user passing arguments via the CLI and specifying a configuration file.
+2. Environment variables common to all ROS packages are declared. In addition, the global arguments common across
+all ROS packages are declared.
+3. For each package launch file:
+    - The CLI arguments, global argument declarations, and environment variables are passed into the package launch
+      file.
+    - Local arguments, specific to the package, are declared. Both the global and local arguments are parsed based on the
+      argument declarations and are set for use upon start up.
+    - The ROS nodes belonging to the package begin execution, utilizing the ROS parameters from the configuration file.
+
+## Invoking Launch Files
+
+??? tip "Stopping the execution of a launch file"
+    Entering ++ctrl+c++ in the terminal where the launch file was invoked will stop all associated
+    ROS packages from running.
+    > Use ++command+c++ for Mac OS
+
+### Package Launch
+
+At the bare minimum, the following packages need to be built with the `Build` or `Build All` VS Code task before launching:
+
+- `custom_interfaces`
```

Comment:
This only has to be built when you first set up the workspace and if you make/pull any changes to the repo